src/rust/engine/testutil/local_cas/src/main.rs

```diff
@@ -0,0 +1,38 @@
+#[macro_use(value_t)]
+extern crate clap;
+extern crate env_logger;
+extern crate mock;
+
+use clap::{App, Arg};
+use mock::StubCAS;
+use std::process::exit;
+
+/// TODO:
```

Comment:
In future, it would be good to make only slightly-related changes in independent PRs. But probably ok to land this here.